exceptional_synchrony.gemspec

```diff
@@ -19,6 +19,6 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'em-synchrony'
   spec.add_dependency 'em-http-request'
   spec.add_dependency 'eventmachine'
-  spec.add_dependency 'exception_handling', '>= 2.10', '< 4.0'
+  spec.add_dependency 'exception_handling', '>= 2.10'
```

Comment:
Can you update the version and changelog since this actually changes the Gem source code.

Reply:
Changed in c33ea7f8b5cc0341bdc630aa2a22debca6e89384